pkgs/os-specific/linux/audit/default.nix

```
@@ -19,8 +20,7 @@ stdenv.mkDerivation rec {
   outputs = [ "bin" "dev" "out" "man" ];
 
   depsBuildBuild = [ buildPackages.stdenv.cc ];
-  nativeBuildInputs = lib.optionals stdenv.hostPlatform.isMusl
-    [ autoconf automake libtool ];
+  nativeBuildInputs = [ autoconf automake libtool autoreconfHook ];
```

```suggestion
  nativeBuildInputs = [ autoreconfHook ];
```
included in the hook